### PR TITLE
Create a table for 2020 Scholarx mentees

### LINKED ISSUE #921: Create a table for 2020 scholarx mentees
**Is your feature request related to a problem? Please describe.**
In the archive section of scholarx2020 a table needs to be created to add the mentees to the website.

**Describe the solution you'd like**
The table should contain an image, name, mentor and the university or a single mentee. When clicked on a single mentee the LinkedIn profile should be redirected. The data file can be found [here](https://docs.google.com/spreadsheets/d/1UquBXrpWIxdLdy5vOeflvtaKUgv6z-GhVMLK-g44gPU/edit#gid=0)

In the sheet scroll down for 2020(cell A24 onwards) The sheet format is as follows.
Column A:- Mentor Name.
Column D:- Students for the mentor.
Column E:- LinkedIn Link of the student.
Column F:- University of the student.

These data should be used when creating the table. The table format can be found [here](https://sefglobal.org/scholarx/archive/2019/)

When a student is selected from the table row his/her LinkedIn profile should be redirected. 
The image of the mentee can be taken from each individual's LinkedIn account.

### PR DESCRIPTION
**Pull request by YaiyaSam**
## Purpose

The purpose of this PR is to fix #921

## Goals

To create a new table for 2020 Scholarx mentees

### Screenshots

image

  
### Preview Link

https://pr-925-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Test environment
 
Windows 10
Google Chrome
Node Server

### CODE REVIEW

File: scholarx/archive/2020/index.html [hunk 1]
```diff
@@ -265,4 +275,49 @@ <h6>{{title}}</h6>
...
+                <img src="images/mentees/{{image}}" class="avatar">
+            </th>
+                <td>{{name}}</td>
+                <td>{{mentor}}</td>
+                <td>{{university}}</td>
+          </tr>
+        {{/data}}
+        </tbody>
+      </table>
+</script>
```
Comment (Gravewalker666) [inline scholarx/archive/2020/index.html]: Let's move the static elements to the above html section and keep only the dynamic part (`table body` and `table rows`) in the template.

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]
Great work @YaiyaSam 
I requested some minor changes =)

File: scholarx/archive/2020/index.html (hunk 1 above)
Comment (YaiyaSam) [reply scholarx/archive/2020/index.html]: Resolved with commit d22dba051262c2910d63438a701b27665923ea2f . Please review

**Review by Gravewalker666 — DISMISSED** [review]
Nice, Thank you for contributing @YaiyaSam 
The code looks fine, @janethavi can you check the mentee data again before merging?

**Review by janethavi — CHANGES_REQUESTED** [review]
Thank you very much for this amazing work @YaiyaSam! You have done a great job. A small request. There is a comment on #920. According to that, there are some small changes that were not mentioned in the issue. It would be great if you can adjust them.

1) To keep consistency only two names of the mentor and mentee should be used.
2) In `Theodore John Kotelawala`'s record we can remove (LSE)
3) In `Arundika Weerasekera`'s record we can remove `Liverpool John Moores University, UK ` and keep the rest.
4) In `Thilina Harshana Dhammage`'s record we can remove `Coventry University/`
5) The column name of 'Image' should be removed. (Keep the column without a column header.)

Sorry for not mentioning these on the issue because these were the suggestions made after the comment received on #920

**Comment by YaiyaSam** [comment]
> Thank you very much for this amazing work @YaiyaSam! You have done a great job. A small request. There is a comment on #920. According to that, there are some small changes that were not mentioned in the issue. It would be great if you can adjust them.
> 
> 1. To keep consistency only two names of the mentor and mentee should be used.
> 2. In `Theodore John Kotelawala`'s record we can remove (LSE)
> 3. In `Arundika Weerasekera`'s record we can remove `Liverpool John Moores University, UK ` and keep the rest.
> 4. In `Thilina Harshana Dhammage`'s record we can remove `Coventry University/`
> 5. The column name of 'Image' should be removed. (Keep the column without a column header.)
> 
> Sorry for not mentioning these on the issue because these were the suggestions made after the comment received on #920

will do the changes : )

**Comment by akshika47** [comment]
@janethavi thank you for following up with the changes; @YaiyaSam looking forward to the updated PR. Good work.

**Comment by YaiyaSam** [comment]
@janethavi Made the requested changes with commit https://github.com/sef-global/sef-site/commit/01930b03bbd8324bbcae7a86eb963b92e2c9ff42 Please review

File: scholarx/archive/2020/mentees.json [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+    {
+        "name": "Visal Senathilaka",
+        "image": "visal.jpg",
+        "mentor": "Nishanta Rajakaruna",
+        "university": "University of Sri Jayawardenapura",
+        "linkedin": "https://www.linkedin.com/in/visal-senathilaka-959bb4145/"
+    },
+
+    {
+        "name": "Theodore John Kotelawala",
```
Comment (janethavi) [inline scholarx/archive/2020/mentees.json]: Will you be able to remove the name `Theodore`? So it will be like `John Kotelawala`

Comment (YaiyaSam) [reply scholarx/archive/2020/mentees.json]: Sorry, I dont know how I missed that one. Fixed it with commit https://github.com/sef-global/sef-site/commit/42964b6d75940fa60b324e12357448b3cd3f1928

File: scholarx/archive/2020/mentees.json [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+        "image": "nadarasar.jpg",
+        "mentor": "Jeewika Ranaweera",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/nadarasar-bahavan/"
+    },
+
+    {
+        "name": "Pathum Goonathilake",
+        "image": "pathum.jpg",
+        "mentor": "Jeewika Ranaweera",
```
Comment (akshika47) [inline scholarx/archive/2020/mentees.json]: It is Dr. Jeewika Ranaweera. Apply for all the entries and please check spellings.

File: scholarx/archive/2020/mentees.json [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+        "image": "suchitha.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/suchitha-dehigaspitiya/"
+    },
+
+    {
+        "name": "Sahan Samuditha",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Nishanta Rajakaruna",
```
Comment (akshika47) [inline scholarx/archive/2020/mentees.json]: It is Prof. Nishanta Rajakaruna. [Apply for all the entries and please check spellings.]

File: scholarx/archive/2020/mentees.json [hunk 5]
```diff
@@ -0,0 +1,426 @@
...
+        "image": "yashodha.jpg",
+        "mentor": "Nushelle De Silva",
+        "university": "General Sir John Kotelawala Defence University",
+        "linkedin": "https://www.linkedin.com/in/yashodha-warakapitiya-094a4015a/"
+    },
+
+    {
+        "name": "Amaya Dharmasiri",
+        "image": "amaya.jpg",
+        "mentor": "Rukshan Batuwita",
```
Comment (akshika47) [inline scholarx/archive/2020/mentees.json]: It is Dr. Rukshan Batuwita. [Apply for all the entries and please check spellings.]

File: scholarx/archive/2020/mentees.json [hunk 6]
```diff
@@ -0,0 +1,426 @@
...
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Sajani Amarasiri",
+        "university": "Sri Lanka Institute of Information Technology",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Chamodi Perera",
+        "image": "chamodi.jpg",
+        "mentor": "Sumudu Herath",
```
Comment (akshika47) [inline scholarx/archive/2020/mentees.json]: It is Dr. Sumudu Herath. [Apply for all the entries and please check spellings.]

File: scholarx/archive/2020/index.html [hunk 7]
```diff
@@ -265,4 +285,37 @@ <h6>{{title}}</h6>
...
 </script>
+
+<script type="text/html" id="templateMentees">
+    {{#data}}
+        <tr onclick="window.location='{{linkedin}}';">
+            <th scope="row">
+                <img src="images/mentees/{{image}}" class="avatar">
+            </th>
+                <td>{{name}}</td>
+                <td>{{mentor}}</td>
```
Comment (akshika47) [inline scholarx/archive/2020/index.html]: I suggest switching the order to be Name University Mentor

File: scholarx/archive/2020/mentees.json [hunk 8]
```diff
@@ -0,0 +1,426 @@
...
+        "university": "General Sir John Kotelawala Defence University",
+        "linkedin": "https://www.linkedin.com/in/pathum-goonathilake-0181b7140/"
+    },
+
+    {
+        "name": "Muraleetharan Mathanraj",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Jeewika Ranaweera",
+        "university": "University of Jaffna",
+        "linkedin": ""
```
Comment (akshika47) [inline scholarx/archive/2020/mentees.json]: https://www.linkedin.com/in/mathanraj-sharma/ 
Please update the name and the headshot as well.

File: scholarx/archive/2020/mentees.json [hunk 9]
```diff
@@ -0,0 +1,426 @@
...
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/anushiya-thevapalan/"
+    },
+
+    {
+        "name": "Miluckshan Jalangan",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Sajani Amarasiri",
+        "university": "Sri Lanka Institute of Information Technology",
+        "linkedin": ""
```
Comment (akshika47) [inline scholarx/archive/2020/mentees.json]: https://www.linkedin.com/in/miluckshan-j/ [Update the photo]

File: scholarx/archive/2020/mentees.json [hunk 10]
```diff
@@ -0,0 +1,426 @@
...
+        "name": "Muraleetharan Mathanraj",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Jeewika Ranaweera",
+        "university": "University of Jaffna",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Shehana Iqbal",
+        "image": "shehana.jpg",
```
Comment (akshika47) [inline scholarx/archive/2020/mentees.json]: This seems to not work. please double check

**Review by akshika47 — COMMENTED** [review]
@janethavi  @YaiyaSam I have added a few minor changes as well!

**Comment by YaiyaSam** [comment]
@janethavi  @akshika47  I made the requested changes with commit https://github.com/sef-global/sef-site/commit/2c99a9159bd708ed92c0930ff84d0ec93ee70ebe Please review

**Review by akshika47 — APPROVED** [review]

**Comment by akshika47** [comment]
@janethavi could you approve this?

**Review by janethavi — APPROVED** [review]
@YaiyaSam Thank you very much! You have done a great job. Great communication and response to all the changes requested.

**Comment by akshika47** [comment]
Yes, agree with @janethavi! Really professional work @YaiyaSam. Keep it up.

**Pull request merged**